.github/workflows/ci.yml

```diff
@@ -564,9 +564,7 @@ jobs:
           - { sample: java-protobuf-eventsourced-shopping-cart, it: true }
 
           - { sample: java-spring-eventsourced-customer-registry, it: true }
-          # FIXME would be good to enable integration tests but currently failing to resolve host.docker.internal
-          # pre_cmd: 'docker-compose -f ../java-spring-eventsourced-customer-registry/docker-compose.yml up -d; (cd ../java-spring-eventsourced-customer-registry/; mvn spring-boot:run &);sleep 10;'
-          - { sample: java-spring-eventsourced-customer-registry-subscriber, it: false }
+          - { sample: java-spring-eventsourced-customer-registry-subscriber, it: true, pre_cmd: 'mvn -f ../java-spring-eventsourced-customer-registry/pom.xml package docker:build' }
```

Comment:
For the integration tests of customer-registry-subscriber, we need to first locally publish the image for customer-registry.